app/assets/javascripts/school_administrator_dashboard/DashboardScatterPlot.js

```diff
@@ -20,7 +20,7 @@ export default class DashboardScatterPlot extends React.Component{
   }
```

Comment:
This seems great!  In this PS, can you also move this file into `components` since we'll re-use it across the `school_administrator_dashboard` and `student_profile` folders?